tensorflow/compiler/xla/stream_executor/dnn.h

```diff
@@ -965,13 +965,22 @@ class OpRunner<void(Args...)> {
   virtual tsl::Status operator()(Stream*, ProfileResult*,
                                  DeviceMemoryBase scratch_memory,
                                  Args... args) const = 0;
+
+  // Launch the operation with a variable number of operands.
+  virtual tsl::Status operator()(Stream*, ProfileResult*,
+                                 DeviceMemoryBase scratch_memory,
+                                 std::vector<DeviceMemoryBase>) const {
+    return tsl::errors::Unimplemented("operator() not implemented.");
+  };
```

Comment:
Instead of creating a second overload, we should just stick with the orginal and have the `Args...` template paramater be an `std::vector<DeviceMemoryBase>`. Meaning, you should do this:

```c++
using GraphConvSignature = void(std::vector<DeviceMemoryBase>);
using GraphConvRunner = OpRunner<GraphConvSignature>;
```

Right now, it's confusing how there's a second overload which ignores the template argumenst and should only be used for GraphConvRunner.